libs/atqr/domain/src/lib/challenge/challenge.entity.ts

```diff
@@ -3,7 +3,7 @@ import { Transform } from 'class-transformer';
 import { dateDiff } from '../../utils/dateDifference';
 import { EmailAddress } from '../EmailService';
 import { PaymentMethodEntity } from '../PaymentMethod';
-import { Player } from '../player/player.entity';
+//import { Player } from '../player/player.entity';
 
 export enum SupervisorEnum {
```

Comment:
are you sure you shouldn't update the properties outside the constructor as well as the exported entity's name?

Reply:
there is a task only for updating the proprieties ! I did it and it is in review. I would prefer it to me all together. But they are separated tasks @rairezende

Reply:
ok!